Java.md

```diff
@@ -9,8 +9,8 @@
 
 Java was conceptualized back in [1991](http://web.archive.org/web/20050420081440/http://java.sun.com/features/1998/05/birthday.html), but it was too advanced for its original purpose of helping broadcast cable news better. Finally, in 1995, first public release of Java 1.0 was made by Sun Microsystems.
 
-This language has been around a long time, and is known for its battle-hardened reliability. The [JVM](https://en.wikipedia.org/wiki/Java_virtual_machine) lets you write platform independent Java code.
+This language has been around a long time, and is known for its battle-hardened reliability. The [JVM](https://en.wikipedia.org/wiki/Java_virtual_machine) lets you write platform independent Java code. This is achieved by first compiling the source code to bytecode, which is later translated to corresponding machine code. The big advantage of this procedure is that bytecode runs inside the JVM, which is responsible for compiling it into matching machine code during runtime. So every time a new operating system comes out, Oracle has to write a JVM and every JAVA application can theoretically run on it. 
```

Comment:
> So every time a new operating system comes out, Oracle has to write a JVM and every JAVA application can theoretically run on it.

That's a funny way to put it! But no, new OS, at least at a kernel level, don't come out that often.